tests/pages/home_pages/test_bip85.py

```diff
@@ -144,6 +144,18 @@ def test_bip85_bip39_mnemonic_derivation(mocker, amigo, tdata):
             "a620a1bc",
             True,
         ),
+        # 8 - Cancel at mnemonic word count selection
+        (
+            tdata.SINGLESIG_12_WORD_KEY,
+            [
+                BUTTON_ENTER,  # BIP39 Mnemonic
+                BUTTON_PAGE_PREV,  # Move to "< Back"
+                BUTTON_ENTER,  # Press "< Back"
+            ],
+            None,  # No mnemonic generated
+            None,  # No fingerprint
```

Comment:
If we do not generate a bip85 mnemonic, we still have a fingerprint from `tdata.SINGLESIG_12_WORD_KEY`:

```suggestion
            "a620a1bc",
```